### PR TITLE
Ft-user-login

### PR DESCRIPTION
**Pull request by sadiqasg**
User login feature

### CODE REVIEW

File: src/config/generateCode.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const generateCode = () => {
```
Comment (forestbaba) [inline src/config/generateCode.ts]: This should rather be in a service. Let all business logic stay in service please

File: src/config/nodemailerConfig.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+const transporter = nodemailer.createTransport({
+    service: "gmail",
+    auth: {
+        user: NODEMAILER_EMAIL,
+        pass: NODEMAILER_PASS,
+    },
+});
+
+export const sendConfirmEMail = async (
```
Comment (forestbaba) [inline src/config/nodemailerConfig.ts]: Can you revamp this to use sendgrid instead of nodemailer?

Comment (forestbaba) [reply src/config/nodemailerConfig.ts]: I will share the credentials later today

Comment (sadiqasg) [reply src/config/nodemailerConfig.ts]: Alright noted. I'll be expecting the credentials

File: src/config/generateCode.ts (hunk 1 above)
Comment (sadiqasg) [reply src/config/generateCode.ts]: Okay, I'm on it.

**Review by forestbaba — APPROVED** [review]

**Pull request merged**